tamr_toolbox/utils/client.py

```diff
@@ -85,6 +90,61 @@ def create(
     return client
 
 
+def create_with_jwt(
+    *,
+    token: str,
+    host: str,
+    port: Optional[Union[str, int]] = 9100,
+    protocol: str = "http",
+    base_path: str = "/api/versioned/v1/",
+    session: Optional[requests.Session] = None,
+    store_auth_cookie: bool = False,
+    enforce_healthy: bool = False,
+) -> Client:
+    """Creates a Tamr client from the provided configuration values using a JWT token instead
+    of a username and password. Note that this feature is only available on v2022.010.0 or later.
+
+    Args:
+        token: A JWT token to authenticate the client
```

Comment:
what is the behaviour if the token is an empty string?

Reply:
The client will still be created. Users will run into issues though when they try to hit an API that requires credentials.